components/builder-originsrv/src/data_store.rs

```diff
@@ -1107,10 +1107,10 @@ impl DataStore {
         let rows = conn.query(
             "SELECT * FROM insert_origin_integration_v1($1, $2, $3, $4)",
             &[
-                &oic.get_origin(),
-                &oic.get_integration(),
-                &oic.get_name(),
-                &oic.get_body(),
+                &oic.get_integration().get_origin(),
+                &oic.get_integration().get_integration(),
```

Comment:
At some point we should fix the overloading of the 'integration' term.  Maybe 'integration' should be 'type' instead.  This can wait for a refactor as there will likely be other things we'll want to do at the same time.

Reply:
Yeah, agreed.